rustls/src/lib.rs

```diff
@@ -533,6 +533,72 @@ pub mod unbuffered {
     };
 }
 
+/// Kernel connection API.
```

Comment:
Nit: I think this docstring should be inside the `conn/kernel.rs` file. Could we just do `pub use conn::kernel;`?